config/locales/lt.yml

```diff
@@ -0,0 +1,49 @@
+lt:
+  language_name: Lithuanian
```

Comment:
Would be good to have Lithuanian language name in Lithuanian
```suggestion
  language_name: Lietuvių 
```

Reply:
Yeah, I thought about it but wasn't sure how you wanted it. Commited your suggestion, thanks!